packages/react/src/index.tsx

```diff
@@ -146,26 +164,33 @@ export function useWithContext(context: Context, parent = useConfidence()): Conf
     [parent, Value.serialize(context)],
   );
 
-  const [, setState] = useState(() => child.contextState);
-  useEffect(
-    () =>
-      child.subscribe(state => {
-        if (state === 'READY' || state === 'ERROR') setState(child.contextState);
-      }),
-    [child, setState],
-  );
   return child;
 }
 
+// this would be better named useContext, but would then collide with React.useContext
+// eslint-disable-next-line react-hooks/rules-of-hooks
+export function useConfidenceContext(confidence = useConfidence()): Context {
```

Comment:
I'm considering if we should put all the hooks as static methods on ConfidenceReact. Then this would become `ConfidenceReact.useContext` and it wouldn't collide with the `React.useContext` hook...

Reply:
Not sure what the standard is... `ConfidenceReact.useContext` does feel a bit strange to me... had it been `Confidence.useContext` would maybe be better but also a bit strange IMO.

Reply:
Let's skip this for now.